pandas/tests/indexes/datetimes/test_ops.py

```diff
@@ -175,6 +175,17 @@ def test_round(self):
             tm.assertRaisesRegexp(ValueError, msg, rng.round, freq='M')
             tm.assertRaisesRegexp(ValueError, msg, elt.round, freq='M')
 
+            # GH 14440
```

Comment:
can you also round to us/ns here as well (which should equal the original)

as an aside, you can now do parametrized tests (but have to move them to separate functions and not class based)